File: python/area_tri.py
# Function to calculate the area of a triangle
def calculate_triangle_area(base, height):
    return 0.5 * base * height

# Input: Enter the base and height of the triangle
base = float(input("Enter the base of the triangle: "))
height = float(input("Enter the height of the triangle: "))

# Calculate the area
area = calculate_triangle_area(base, height)

# Output: Print the area of the triangle
print(f"The area of the triangle is: {area}")

File: C/area_tri.c
// Area of Triangle
#include <stdio.h>

// Function to calculate the area of a triangle
double calculate_triangle_area(double base, double height) {
    return 0.5 * base * height;
}

int main() {
    double base, height, area;

    // Input: Enter the base and height of the triangle
    printf("Enter the base of the triangle: ");
    scanf("%lf", &base);

    printf("Enter the height of the triangle: ");
    scanf("%lf", &height);

    // Calculate the area
    area = calculate_triangle_area(base, height);

    // Output: Print the area of the triangle
    printf("The area of the triangle is: %.2f\n", area);

    return 0;
}
